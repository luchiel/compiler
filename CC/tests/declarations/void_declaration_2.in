typedef void f;
void main()
{
f a;
}