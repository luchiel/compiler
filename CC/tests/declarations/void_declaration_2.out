Error: ParseError:4:4 Variable declared as void
